### DISCUSSION TITLE
Can I use this to play j2me on ios 3?

### DISCUSSION THREAD

**Discussion by Baggette** [SquirrelJME/SquirrelJME#141] [General Questions] [answered]
Sorry if I didn't follow the format correctly im not sure how I was supposed to do it.

**Comment by Baggette** [comment]
nvm

**Comment by SergioFLS** [comment]
I think I've seen a port of some third-party JVM on Cydia. Not sure if it supports iOS 3 though

**Comment by Baggette** [comment]
> I think I've seen a port of some third-party JVM on Cydia. Not sure if it supports iOS 3 though

there is a third party jvm on cydia that does support ios 3

**Comment by Baggette** [comment]
I don't know what to do with it though

**Comment by XerTheSquirrel** [comment, marked as answer]
Hi! iOS 3 probably not as it is very old and RetroArch only supports iOS 9 and up.

**Reply by Baggette** [reply]
> Hi! iOS 3 probably not as it is very old and RetroArch only supports iOS 9 and up.

I understand I was just fooling around with my old ipod touch 1g and was wondering if there was any j2me emulators, im sure the hardware can handle it. I already have a few emulators on it (snes. gba, nes, n64)

**Reply by XerTheSquirrel** [reply]
It would be able to handle it fine, it mostly is the matter of porting to the platform. I do not have any iOS devices on hand.

**Reply by Baggette** [reply]
> It would be able to handle it fine, it mostly is the matter of porting to the platform. I do not have any iOS devices on hand.

you can pick up touch 1's and stuff pretty cheap on ebay
